.github/workflows/check.yml

```
@@ -39,3 +39,15 @@ jobs:
 
       - name: Rustfmt
         run: cargo +nightly fmt --all -- --check
+
+  stylua:
+    name: stylua
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: JohnnyMorganz/stylua-action@v3
```

```suggestion
      - uses: actions/checkout@v4
      - uses: JohnnyMorganz/stylua-action@v4
```

Any reason not to use `v4`?

---

Hmm not really, let's use `v4` instead 👍🏻